homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -703,3 +950,88 @@ async def async_browse_media(
             media_content_id,
             content_filter=lambda item: item.media_content_type.startswith("audio/"),
         )
+
+    async def async_join_players(self, group_members: list[str]) -> None:
+        """Create a Beolink session with defined group members."""
+
+        # Use the touch to join if no entities have been defined
```

Comment:
What does "the touch to join" refer to?

Reply:
Touch to join is the term used joining a Beolink Session on a Bang & Olufsen device by interacting with the product https://support.bang-olufsen.com/hc/en-us/articles/4411572885777-How-does-my-speaker-Join-an-ongoing-experience. Touch to join here refers to the command that triggers this same behavior.